components/avatar/avatar.css

```diff
@@ -1,25 +1,26 @@
 @import url("../../default.css");
 
+.av{
```

Comment:
Hey Saran avatar will be the more informative class name rather than av which doesn't convey much meaning. It will make the code more readable.

Reply:
Hi Mahesh, Thanks for reviewing, I will refactor the card component.